.oscal/poam.oscal.yaml

```
@@ -0,0 +1,308 @@
+
+  risks:
+    - uuid: 77777777-0000-4000-9000-100000000014
+      title: "RISK-1: Inadequate Security Requirements Management"
+      description: "Security requirements are not properly documented and maintained, increasing risk of security gaps"
+      statement: >-
+        Lack of centralized security requirements documentation and systematic review
+        process could lead to inconsistent security implementations and missed
+        vulnerabilities. This poses significant risk to the development environment
+        and resulting software products.
+      props:
+      props:
```

yamllint (1.35.1)

[error] duplication of key "props" in mapping

(key-duplicates)